lib/osml/utils/regional_configs.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates.
+ */
+
+import { Runtime } from "aws-cdk-lib/aws-lambda";
+
+import { RegionConfig } from "./regional_config";
+
+export const defaultConfig = {
+  ecrCdkDeployRuntime: Runtime.PROVIDED_AL2023,
+  maxVpcAzs: 3,
+  sageMakerEndpointInstanceType: "ml.p3.2xlarge"
```

Comment:
Could we use `ml.p3.2xlarge` as a default GPU instance type in `MRDataplane` class? OR  update this to be something like `sageMakerGpuEndpointInstaceType` because we have a CPU variable initialized in MRDataplane. I think I am leaning towards the former rather than the latter.

Reply:
Well it is the default. We just store it in the config rather than the class now. Would you prefer we store the default in the class? It just doesn't really match the pattern we use for the other config values such as maxVpcAzs and ecrCdkDeployRuntime.

Reply:
It somewhat matches the pattern in `MRDataplaneConfig` if we look at Line 51 in `MRDataplane` class, CPU vs GPU.

Reply:
Sure but it doesn't really make sense to store a default in both the config and the class. The patter for `maxVpcAzs` and `ecrCdkDeployRuntime` is that the customer can specify the property if they want, but if undefined, we pull from the regionalConfig.

If we were to implement your suggestion for `sageMakerEndpointInstanceType`, we would have to flip this logic. If the the property is defined in the regionalConfig use it, otherwise use the default value in the class.

I am inclined to leave it but rename the config to `sageMakerGpuEndpointInstaceType`.

Reply:
Well, remove `sageMakerEndpointInstaceType` property entirely from this package (this is what I meant by) and use the standard GPU instance that works in all regions.

If you do not prefer this route, then I am fine with renaming that property to be more specific type (CPU vs GPU): `sageMakerGpuEndpointInstaceType` so a user can understand if they are passing as GPU instance or CPU instance.

Reply:
Ohhhh I am sorry - your point makes sense now. I think the main problem is we assumed `ml.p3.2xlarge` was available in all regions but turned out it wasn't in us-west-1 or us-gov-east-1. 

Even if we do choose an instance type that is currently available in all regions, we can't assume it will be available in all new regions. I think we'll just eventually run into the same problem.